apps/client/src/common/models/OntimeSettings.ts

```diff
@@ -8,4 +8,5 @@ export const ontimePlaceholderSettings: Settings = {
   operatorKey: null,
   timeFormat: '24',
   language: 'en',
+  apiAllowTimeChange: false,
```

Comment:
hmmm, not super eager on this setting

I think changing too much behaviour on settings is a bad practice.
In my opinion, other software that does this ends up being difficult to use and to predict the behaviour of.
You get the situations where there are 5 technicians crunched over a laptops going through the multiple settings that affect something to try and explain some unexpected behaviour.

I think apps (and Ontime) should have reasonable standards.
We can choose to allow this or not, but that should be it.

The issue that I see, is that a bad integration (ie: someone using a module or a script which has issues) could make Ontime unusable.
The user would not see the connection since the problem surfaces in Ontime. Our user base tends to be unforgiving with runtime crashes 😅 

Do we have an expectation on how much of the users this whitelisting would improve the lives of?
Maybe the way forwards is to whitelist everything and do some rate limiting?